From 28f51890f8c0ec3ec02a6f836e8825ca67374259 Mon Sep 17 00:00:00 2001
From: David <dmck2b@gmail.com>
Date: Wed, 23 Apr 2014 01:01:47 +0100
Subject: [PATCH] Stop anvils from destroying all items in the second slot

While this may allow multiple stacked items to be repaired at once, it's better than eating all of a user's items.

diff --git a/src/main/java/net/minecraft/server/ContainerAnvilInventory.java b/src/main/java/net/minecraft/server/ContainerAnvilInventory.java
index 1afa6e7..6d339e5 100644
--- a/src/main/java/net/minecraft/server/ContainerAnvilInventory.java
+++ b/src/main/java/net/minecraft/server/ContainerAnvilInventory.java
@@ -43,7 +43,7 @@ public class ContainerAnvilInventory extends InventorySubcontainer { // CraftBuk
     ContainerAnvilInventory(ContainerAnvil containeranvil, String s, boolean flag, int i) {
         super(s, flag, i);
         this.a = containeranvil;
-        this.setMaxStackSize(1); // CraftBukkit
+        this.setMaxStackSize(64); // CraftBukkit
     }
 
     // CraftBukkit start - override inherited maxStack from InventorySubcontainer
-- 
1.8.3.2

